locations/spiders/medical_city_healthcare.py

```diff
@@ -16,52 +15,38 @@ class MedicalCityHealthcareSpider(scrapy.Spider):
+        ("HOSPITAL - GENERAL", Categories.HOSPITAL),
+        ("HOSPITAL - CHILDRENS", Categories.HOSPITAL),
+        ("BEHAVIORAL DEPARTMENT", {"healthcare": "psychotherapist", "healthcare:speciality": "behavior"}),
+        ("HOSPITAL - PSYCHIATRIC", {"healthcare": "psychotherapist", "healthcare:speciality": "behavior"}),
         (
-            "Imaging_Center",
+            "IMAGING CENTER",
             {"amenity": "hospital", "healthcare": "hospital", "healthcare:speciality": "diagnostic_radiology"},
         ),
-        ("Surgery_Center", {"amenity": "hospital", "healthcare": "hospital", "healthcare:speciality": "surgery"}),
-        ("Rehabilitation_Center", {"healthcare": "rehabilitation"}),
-        ("Urgent_Care", Categories.CLINIC_URGENT),
-        ("Cancer_Center", {"healthcare": "centre", "healthcare:speciality": "oncology"}),
-        ("Health_Center", {"healthcare": "centre"}),
-        ("Diagnostic_Center", {"healthcare": "centre"}),
-        ("Womens_Center", {"healthcare": "centre"}),
-        ("Medical_Group", {"healthcare": "centre"}),
-        ("Other", {"healthcare": "centre"}),
+        ("SURGERY CENTER", {"amenity": "hospital", "healthcare": "hospital", "healthcare:speciality": "surgery"}),
+        ("HOSPITAL - REHABILITATION", {"healthcare": "rehabilitation"}),
+        ("URGENT CARE CENTER", Categories.CLINIC_URGENT),
+        ("CANCER CENTER", {"healthcare": "centre", "healthcare:speciality": "oncology"}),
+        ("HEALTH CENTER", {"healthcare": "centre"}),
     ]
 
     def parse(self, response):
-        script = response.xpath('//script[contains(text(), "hostLocations")]').extract_first()
-        data = re.search(r"var hostLocations = (.*]);", script).group(1)
-        locations = json.loads(data)
-
-        for location in locations:
-            properties = {
-                "ref": location["id"],
-                "name": location["title"],
-                "street_address": location["address1"],
-                "city": location["city"],
-                "state": location["state"],
-                "postcode": location["zip"],
-                "lat": location["lat"],
-                "lon": location["lng"],
-                "phone": location["phone"],
-                "website": response.url,
-            }
-
-            types = location.get("type")[1:-1].split(",")
+        script = json.loads(response.xpath('//script[@type="application/json"]/text()').get())
+        for data in script["sitecore"]["route"]["placeholders"]["body"][0]["placeholders"]["col-top"][0]["fields"][
+            "defaultResponse"
+        ]["results"]:
+            item = DictParser.parse(data)
+            item["name"] = data.get("facilityName")
+            item["phone"] = data.get("marketedPrimaryPhoneNumber")
+            item["website"] = data.get("marketedWebsite")
+            item["street_address"] = data.get("street1")
+            types = data.get("purposeTypeName")
+            item["street"] = types
```

Comment:
`street` is getting things like "URGENT CARE CENTER". That should probably be used to set tagging, not to set street name.